### DISCUSSION TITLE
Improvement of UI/UX contribution grid.

### DISCUSSION THREAD

**Discussion by mar1n** [community/community#144977] [Other Feature Feedback, Questions, & Ideas]
Hi,
when I want to see details from day(square) above currently highlighted day(square)  in contribution screen the pop up is blocking space of day(square) above so I need to move cursor back and forth to see it.
When we move cursor horizontally is alright but when we move vertically the pop up is blocking square under your self so we need to move cursor back and forth.
![github_UI](https://github.com/user-attachments/assets/2e748e44-b2a7-4c26-b579-f075f4e16e76)

**Comment by dalvarez2596** [comment]
Hi @mar1n 
Yeah, that's true. I just noticed it too. When you go from top to bottom, there's no problem, but if you try going from bottom to top, the popup blocks it.

Also, I think you should add the label "Product Feedback", I see this more as a feedback to the github team.

**Comment by mar1n** [comment]
I don't how can I add a label " Product Feedback".

**Reply by dalvarez2596** [reply]
I read something about it and this should work :

First you need to open the discussion.
Then you go to the labels section and should look for the gear icon(⚙️) next to "Labels" on the right side.
After that you the labels menu should appear and once there you can add new labels by selecting them from the list.
You can also remove by clicking the 'x' next to them
Changes are saved automatically

**Reply by mar1n** [reply]
I don't see there  labels I can only change category.
I don't have idea how to add label or remove it.

**Reply by dalvarez2596** [reply]
I think you should be able base on this guide https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-a-label.
If you still not able, you can changing the category.

**Reply by mar1n** [reply]
Yes I change category but I still can't add label.
When I create new discussion and I choose category General there is options to add label.

I've changed category to general but still I don't see option for changing label.
